### PR TITLE
Made field labels translatable

### PR DESCRIPTION
**Pull request by andrasguseo**

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -40,6 +40,10 @@ tends to be very limited.
 
 == Changelog ==
 
+= 1.0.2 2019-10-29 =
```
Comment (cliffordp) [inline readme.txt]: add brackets around all version numbers to match our main plugins' formatting

File: readme.txt [hunk 2]
```diff
@@ -5,7 +5,7 @@ Tags: events, calendar
 Requires at least: 4.7.0
 Tested up to: 4.9.4
```
Comment (cliffordp) [inline readme.txt]: update `Tested up to` and `Requires PHP` versions

**Review by cliffordp — CHANGES_REQUESTED** [review]

File: readme.txt (hunk 1 above)
Comment (andrasguseo) [reply readme.txt]: Should be ok now.

File: readme.txt (hunk 2 above)
Comment (andrasguseo) [reply readme.txt]: Should be ok now as well.

**Review by cliffordp — APPROVED** [review]
if you want, you can put the central ticket ID in brackets at the end of each changelog item

**Pull request merged**